tests/integration/errors/test_error_linking_dt_on_cat_off.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should add transaction GUID to error traces when DT is enabled.
+*/
+
+/*SKIPIF
+<?php
+if (!function_exists('newrelic_get_error_json')) {
+  die("warn: release builds of the agent do not include newrelic_get_error_json()");
+}
+if (!function_exists('newrelic_get_transaction_guid')) {
+  die("warn: release builds of the agent do not include newrelic_get_transaction_guid()");
+}
+*/
+
+/*INI
+newrelic.distributed_tracing_enabled=true
+newrelic.cross_application_tracer.enabled=false
+*/
+
+/*EXPECT_TRACED_ERRORS
+[
+  "?? agent run id",
+  [
+    [
+      "?? when",
+      "OtherTransaction/php__FILE__",
+      "Noticed exception 'RuntimeException' with message 'Hi!' in __FILE__:??",
+      "RuntimeException",
+      {
+        "stack_trace": [
+          " in throw_it called at __FILE__ (??)"
+        ],
+        "agentAttributes": "??",
+        "intrinsics": "??"
+      },
+      "?? txn guid"
+    ]
+  ]
+]
+*/
+
+require_once(realpath (dirname ( __FILE__ )) . '/../../include/tap.php');
+
+function throw_it() {
+  newrelic_notice_error(new RuntimeException('Hi!'));
+}
+
+throw_it();
+
+/* capture error trace json and verify transaction GUID */
+$json = newrelic_get_error_json();
+$payload = json_decode($json, true);
+$guid = newrelic_get_transaction_guid();
+tap_equal($guid, $payload[5], "Error trace includes txn guid");
```

Comment:
Doesn't this require corresponding `/*EXPECT Error trace includes txn guid - ok*/`?

Reply:
Yep - just discovered this and had this commit already:
[7eabda6](https://github.com/newrelic/newrelic-php-agent/pull/812/commits/7eabda6754030cb3fe1f8b4dbf424d50273fa237)